src/identity/components/GlobalHeader/IdentityUserAvatar.tsx

```diff
@@ -100,18 +100,17 @@ class IdentityUserAvatar extends React.Component<Props, State> {
         <div>
           {/* Button shape is ButtonShape.Square to make the height and width the same
             so we can use the border radius to make it a circle  */}
-          {this.getInitials() && (
-            <Button
-              text={this.getInitials()}
-              shape={ButtonShape.Square}
-              color={
-                isPopoverOpen ? ComponentColor.Default : ComponentColor.Tertiary
-              }
-              onClick={this.togglePopoverState}
-              className={userAvatarButtonClassName}
-              testID="global-header--user-avatar"
-            />
-          )}
+          <Button
+            text={this.getInitials()}
+            shape={ButtonShape.Square}
+            color={
+              isPopoverOpen ? ComponentColor.Default : ComponentColor.Tertiary
+            }
+            onClick={this.togglePopoverState}
+            className={userAvatarButtonClassName}
+            testID="global-header--user-avatar"
+            icon={!this.getInitials() ? IconFont.User : null}
```

Comment:
Looks good to me.

For context for others here: the issue here is that when a user doesn't have a first name or last name, we don't want the user avatar to blow up. In Clockface, `<Button />` throws an error unless at least one of the `text` and `icon` properties is truthy.

```
if (!icon && !text) {
      throw new Error('Button requires either "text" or "icon" props')
}
```
No such error is possible here. In the usual case where a user has a first name, `text` will be truthy because `getInitials()` will be truthy. 

In the edge case where the user doesn't have a name - i.e., if `getInitials()` evaluates to empty string (the default result from redux state) or another falsy value, `text` will be `falsy`, making `icon` truthy and setting `icon` to the default profile picture.